windows/CommonSettings.props.example

```diff
@@ -64,6 +72,11 @@
             <PreprocessorDefinitions>WITH_PYTHON_LAYER;BOOST_PYTHON_STATIC_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
         </ClCompile>
     </ItemDefinitionGroup>
+    <ItemDefinitionGroup Condition="'$(MatlabSupport)'=='true'">
+        <ClCompile>
+            <PreprocessorDefinitions>MATLAB_MEX_FILE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
I think it's not necessary, I added it because I'm not sure. 
I found it when I using MATLAB `mex` to compile `.cpp` files. I think the default MATLAB setting is:
`CMDLINE100 : cl /c /Zp8 /GR /W3 /EHs /nologo /MD /O2 /Oy- /DNDEBUG /D_CRT_SECURE_NO_DEPRECATE /D_SCL_SECURE_NO_DEPRECATE /D_SECURE_SCL=0   /DMATLAB_MEX_FILE -Iinclude  -I"C:\Program Files\MATLAB\R2015aSP1\extern\include" ... `

Without these parameters we can also get valid `caffe_.mexw64`.